README.md

```diff
@@ -1,20 +1,43 @@
-# Oam uploader
+<h1 align="center">Uploader Form</h1>
 
-Oam uploader
+<div align="center">
+  <h3>
+  <a href="https://docs.openaerialmap.org/uploader/getting-started">Ecosystem</a>
+  <span> | </span>
+  <a href="https://github.com/hotosm/oam-uploader-api">Uploader API</a>
+  <span> | </span>
+  <a href="https://github.com/hotosm/oam-uploader-admin">Token Manager</a>
+  </h3>
+</div>
+
+The Uploader Interface allows users to upload imagery to be processed and catalogged by the [OpenAerialMap Catalog](https://docs.openaerialmap.org/catalog/) through a form. This web application requires the [Uploader API](https://github.com/hotosm/oam-uploader-api) to be running, and requires a token issued by the [Token Manager](https://github.com/hotosm/oam-uploader-admin). Before proceeding, we suggest you read the ecosystem docs.
```

Comment:
We can write it to say OIN spec instead of OAM catalog; I wonder though if it will make the uploader's purpose too abstract (instead of tying it to the rest of the ecosystem)